src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunction.java

```diff
@@ -30,7 +30,7 @@ public X getValue() {
         double a = firstBound.getValue().getReal();
         double b = secondBound.getValue().getReal();
 
-        if (v >= Math.min(a, b) && v <= Math.max(a, b)) {
+        if (v >= Math.min(a, b) && v < Math.max(a, b)) {
             return DFFactory.get(1d);
```

Comment:
Indeed it's not in the scope, but this is the most convenient way we found to do:
`42 * between(2,1,3) = 42*1 = 42`
The most frequently, the meaning of comparing functions is to confirm or cancel a (mathematical) product.